frontend/public/components/error.tsx

```diff
@@ -63,31 +62,35 @@ export const ErrorPage404: React.SFC<ErrorPage404Props> = (props) => <div>
   <ErrorComponent title="404: Page Not Found" message={props.message} errMessage={props.errMessage} />
 </div>;
 
-export const ErrorBoundaryFallback: React.SFC<ErrorBoundaryFallbackProps> = (props) =>
-  <div className="co-m-pane__body">
-    <h1 className="co-m-pane__heading co-m-pane__heading--center">Oh no! Something went wrong.</h1>
-    <div className="row">
-      <ExpandCollapse textCollapsed="Show Details" textExpanded="Hide Details" bordered={false}>
-        <h3 className="co-section-heading-tertiary">{props.title}</h3>
-        <div className="form-group">
-          <label htmlFor="description">Description: </label>
-          <p>{props.errorMessage}</p>
-        </div>
-        <div className="form-group">
-          <label htmlFor="componentTrace">Component Trace: </label>
-          <div className="co-copy-to-clipboard__stacktrace-width-height">
-            <CopyToClipboard value={props.componentStack.trim()} />
+export const ErrorBoundaryFallback: React.SFC<ErrorBoundaryFallbackProps> = (props) => {
+
+  return (
```

Comment:
There shouldn't be a return. I wouldn't reformat the code at this point since `public` dir doesn't enforce prettier. You can just make the small change and later we will sweep the code for formatting changes.

Reply:
Done